package.json

```diff
@@ -23,14 +24,15 @@
   "homepage": "https://github.com/AndreyBelym/nodejs-katas#readme",
   "devDependencies": {
     "chai": "^4.2.0",
+    "cross-env": "^7.0.0",
+    "debug": "^4.1.1",
```

Comment:
Take a closer look: you use `debug` in the production code (in your server). Should we really move it to the `devDependencies`?